src/main/java/com/google/api/codegen/transformer/go/GoSurfaceNamer.java

```diff
@@ -441,7 +442,11 @@ public String getSmokeTestClassName(InterfaceConfig interfaceConfig) {
 
   @Override
   public String injectRandomStringGeneratorCode(String randomString) {
-    return randomString.replace(
-        InitFieldConfig.RANDOM_TOKEN, "\" + strconv.FormatInt(time.Now().UnixNano(), 10) + \"");
+    randomString =
+        CommonRenderingUtil.stripQuotes(randomString)
+            .replace(
+                InitFieldConfig.RANDOM_TOKEN,
+                "\" + strconv.FormatInt(time.Now().UnixNano(), 10) + \"");
+    return "\"" + randomString + "\"";
```

Comment:
If the configs haven't been ported to have quotes RHSs (as I gather from your TODO in the other file), will this cause too many quotes to occur right now?

Reply:
No because we `stripQuotes` on L446 right?

Reply:
And if something didn't have quotes, adding them here should be safe, correct? OK.